src/OpenTelemetry/Internal/HttpSemanticConventionHelper.cs

```diff
@@ -27,8 +32,10 @@ namespace OpenTelemetry.Internal;
 /// </remarks>
 internal static class HttpSemanticConventionHelper
 {
+    public const string SemanticConventionOptInKeyName = "OTEL_SEMCONV_STABILITY_OPT_IN";
```

Comment:
Move this file to [src/Shared](https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/src/Shared) instead.

Reply:
done